src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -68,101 +70,89 @@ g.test('visibility_and_dynamic_offsets')
     }, !success);
   });
 
-g.test('min_buffer_binding_size')
+g.test('non_required_parameter_combined_with_type_separately')
```

Comment:
Name is kind of long but it doesn't make it that clear to me what the test is about. Maybe `binding-type-specific_optional_members`? (I don't remember if hyphens are allowed, but if not then now seems like a good time to add them to the allowed characters list)

Reply:
hyphens is not supported yet. I tried to add "-" or "\-" into src/common/framework/query/validQueryPart.ts, but it doesn't work... Sorry for that.

Reply:
- or \-

Reply:
np, let's forget about the hyphens; the camelCase thing looks good.